crmsh/sbd.py

```diff
@@ -241,6 +248,28 @@ def _enable_sbd_service(self):
             # in init process
             bootstrap.invoke("systemctl enable sbd.service")
 
+    def _adjust_systemd(self):
+        """
+        Adjust start timeout for sbd when has profiles data
+        """
+        if not self._context.profiles_dict:
```

Comment:
The condition has very very weak relationship with `profiles_dict`. And the logic must not be coupled with cloud directly. In general, it might happens to other virtualization environment.

Instead, the condition checks if SBD_DELAY_START configuration is 'no', and the function return if so. And, this is the advanced step after _update_configuration() to enable SBD_DELAY_START in virtualization environment.

Reply:
OK, changed when SBD_DELAY_START set

Reply:
@zzhou1 @gao-yan In virtualization environment but not in Azure, what the value should be for sbd msgwait? Keep default or seem with Azure?